article/sections/_8_conclusion.tex

```diff
@@ -1,3 +1,42 @@
 \section{Conclusion}
+\label{sec:conclusion}
 
-\blindtext
\ No newline at end of file
+Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 
+To our surprise, there has been very little mention of causal inference in the field of travel demand modeling.
+\citet{brathwaite_2018_causal} have documented this disconnect and presented an initial framework for addressing it.
+
+In this chapter, we build up on \citet{brathwaite_2018_causal} and highlight the importance of using causal graphs and 
+causal inference methods in transportation demand modeling efforts.
+
+We presented a numerical exercise aiming to show the importance of  the data generating process in the estimation 
+of effects of interest resulting from external interventions. 
+We showed, using the same outcome model, that different assumptions about the data generating processes could 
+lead to bias in estimates of the effect of interest.
+
+Data generating processes can be illustrated effectively using DAGs as shown by \citet{pearl_1995_causal}.
+We have shown that DAGs help researchers and practitioners represent their assumptions about the data generating process. 
+
+We presented a process allowing researchers and practitioners to construct causal graphs based on their expert opinion of the problem at hand. 
+We also presented methods for testing the implications encoded in the the practitioners proposed causal graph. 
+These implications might be either observed, latent, or both.
+
+Beyond testing the implications and assumptions encoded in a one's causal graph, we describe why causal discovery is important in discerning relationships between covariates one's data.
+We shortly presented several causal discovery algorithms used by
+other researchers in different fields. 
+We highlighted the use of one of these causal discovery algorithms on a simple example. 
+We showed that causal discovery methods help us test independence within our data, 
+develop a clearer picture about the uncertainty within the variables shown in the causal graph, 
+and expand the level and amount of tests performed on our data when compared to the tests implicated 
+by the structure of the researcher's proposed causal graph.
```

Comment:
I find this sentence to be quite complex! There's the list of three things but your third element is itself a complex, compound thought: "expand the level **and** amount of tests [...] when compared to [...]." This is a lot! I would suggest splitting this sentence, or at least splitting out the third element of the sentence.

Second, what's the difference between "help us test independence within our data" and "expand the level and amount of tests [...]." What additional testing help are your referring to? Also what's the level of the tests?

Lastly, I'd say the three biggest benefits of causal discovery are: better characterized uncertainty about one's causal graph, increased understanding about how one's priors on the graph structure affects one's inferences, and increased scrutiny / alteration of one's own graph in reaction to the discovered graphs. No particular order of importance, or at least I don't have strong feelings about the order.